CHANGELOG.md

```diff
@@ -88,6 +88,20 @@ Main (unreleased)
   - Added a new `http_headers` argument to many `discovery` and `prometheus` components.
   - Added a new `scrape_failure_log_file` argument to `prometheus.scrape`.
 
+- Upgrading to OpenTelemetry Collector v0.122
+  - The telemetry level of Otel components is no longer configurable. The `level` argument in the `debug_metrics` block is kept to avoid breaking changes but it is not used anymore.
```

Comment:
Call this (and all other breaking changes) out in the breaking changes section? IMO people need to see this so we should make some noise about it.